frontend/styles/Home.module.css

```diff
@@ -1,116 +1,1067 @@
+  font-family: "Montserrat", sans-serif;
+  font-size: 4.5vw;
+  padding-top: 0px;
+  font-weight: 600;
+  letter-spacing: 4px;
+  line-height: 100px;
+  text-align: right;
+}
+
+/*Description right block*/
+.descriptionRight {
+  line-height: 22px;
+  padding-top: 60px;
+  grid-area: auto / span 1;
+}
+
+/*Right block text*/
+.descriptionRight p {
+  color: var(--general-text);
+  font-family: "Montserrat", sans-serif;
+  font-size: 11pt;
+  text-align: justify;
+}
+
+/*Description center block*/
+.descriptionMiddle {
+  grid-area: auto / span 1;
+}
+
+/*Dark mode*/
+@media (prefers-color-scheme:dark) {
+  .descriptionRight p {
+    color:white;
+  }
+}
+
+/*---------------------------Home page statistics-----------------------------*/
+
+/*Background image*/
+.statsBackground {
+  height: 300px;
+  width: 100%;
+  object-fit: cover;
 }
 
-.footer a {
+.stats {
+  position: relative;
+  overflow: hidden;
   display: flex;
-  justify-content: center;
   align-items: center;
-  flex-grow: 1;
+  justify-content: center;
 }
 
-.title a {
-  color: #0070f3;
-  text-decoration: none;
+.number {
+  display: inline-grid;
+  grid-template-columns: 30% 30% 45%;
+  left: 0;
+  margin: 0 auto;
+  place-items: center center;
+  position: absolute;
+  right: 0;
+  text-align: center;
+  width: 60%;
+  bottom: 30%;
 }
 
-.title a:hover,
-.title a:focus,
-.title a:active {
-  text-decoration: underline;
+.number span {
+  color: var(--yellow);
+  font-family: "Montserrat", sans-serif;
+  font-size: 75pt;
+  font-weight: 500;
+  letter-spacing: 15px;
 }
 
-.title {
-  margin: 0;
-  line-height: 1.15;
-  font-size: 4rem;
+.number p {
+  color: var(--white);
+  font-family: "Montserrat", sans-serif;
+  font-size: 14pt;
 }
 
-.title,
-.description {
+/*-------------------------------Home page events-----------------------------*/
+/*Events block*/
+.events {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  grid-template-rows: 100px auto;
+  grid-column-gap: 50px;
+  padding: 100px 0px;
+  max-width: 85%;
+  margin: 0 auto;
+}
+
+/*Events description grid area*/
+.eventsDescription {
+  width: 100%;
+  height: 100%;
+  padding-bottom: 80px;
+}
+
+/*Events description text*/
+.eventsDescription p {
+  color: var(--general-text);
+  font-family: "Montserrat", sans-serif;
+  font-size: 11pt;
+  text-align: right;
+  padding-top:30px;
+}
+
+.eventsDescription p#about {
+  padding-bottom: 0;
+}
+
+.eventsDescription a {
+  font-family: "Montserrat", sans-serif;
+  text-align: right;
+  color: var(--red);
+  float: left;
+}
+
+/*Events title grid area*/
+.eventsTitle {
+  text-align: left;
+  grid-area: 1 / 2;
+}
+
+.eventsTitle h1 {
+  font-family: "Montserrat", sans-serif;
+  font-size: 55px;
+  text-align: left;
+}
+
+/*Big event image grid area*/
+.event1 {
+  height: 100;
+}
+
+/*Big event image size*/
+.event1 Image {
+  object-fit: contain;
+  width: 100%;
+  height: 100%;
+  transition: transform 0.5s ease;
+  overflow: hidden;
+}
+
+.event1 Image:hover {
```

Comment:
Pretty sure `Image:hover` doesn't do anything in this case, similar with line 252. Think about how we can add hover styles to our images without using the `next/image` tag